src/github.com/couchbase/sync_gateway/base/bucket.go

```diff
@@ -325,7 +330,7 @@ func (bucket CouchbaseBucket) CBSVersion() (major uint64, minor uint64, micro st
 }
 
 // Creates a Bucket that talks to a real live Couchbase server.
-func GetCouchbaseBucket(spec BucketSpec) (bucket Bucket, err error) {
+func GetCouchbaseBucket(spec BucketSpec, callback sgbucket.BucketNotifyFn) (bucket Bucket, err error) {
```

Comment:
Similar question here - is there a reason to not store the callback as part of the BucketSpec?